src/status_im/models/browser-history.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im.models.browser-history
+  (:require [re-frame.core :as re-frame]))
+
+(defn dont-store-history-on-nav-change? [db]
+  (get-in db [:browser/options :dont-store-history-on-nav-change?]))
+
+(defn dont-store-history-on-nav-change []
+  (re-frame/dispatch [:update-browser-options {:dont-store-history-on-nav-change? true}]))
+
+(defn clear-dont-store-history-on-nav-change []
+  (re-frame/dispatch [:update-browser-options {:dont-store-history-on-nav-change? false}]))
+
+(defn dont-store-history-on-nav-change-if-history-exists [db browser-id]
+  (let [browsers (get-in db [:browser/browsers])
+        browser (get browsers browser-id)]
+    (hash-map :dont-store-history-on-nav-change? (some? (:history browser)))))
+
+(defn back [browser]
+  (let [back-index (dec (:history-index browser))
+        back-url (nth (:history browser) back-index)]
+    (dont-store-history-on-nav-change)
```

Comment:
better `dispatch-dont-store-history-on-nav-change`

Reply:
I went with using the bang ie. `dont-store-history-on-nav-change!` because I saw it first as I scanned down the latest comments. :beers: